zipline/assets/_assets.pyx

```diff
@@ -37,12 +37,6 @@ cimport numpy as np
 from trading_calendars import get_calendar
 
 
-# IMPORTANT NOTE: You must change this template if you change
```

Comment:
Was this just not true?

Reply:
There is nothing that references this that I could find in zipline or our downstream consumers